packages/apps-config/src/api/spec/peaq-type-definitions.d.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright 2017-2023 @polkadot/apps-config authors & contributors
```

Comment:
Please remove this - rather add explicit overrides inside the actual file used.

(The rest of these were removed yesterday - don't want to go down the rabbit hole of providing types for these packages where it should be done upstream)

Reply:
Thanks for the feedback. Already removed this file.